cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/upgrade/TemplateConfigurationUpgrader.java

```diff
@@ -20,26 +22,24 @@
 /**
  * This class encompasses all logic for legacy templates configuration detection and upgrading these to the latest
  * supported version.
- *
- * @author mbrunnli (Jun 22, 2015)
  */
 public class TemplateConfigurationUpgrader extends AbstractConfigurationUpgrader<TemplatesConfigurationVersion> {
 
   /**
    * Creates a new {@link TemplateConfigurationUpgrader} instance.
-   *
-   * @author mbrunnli (Jun 23, 2015)
    */
   public TemplateConfigurationUpgrader() {
 
-    super(TemplatesConfigurationVersion.v1_0, TemplatesConfiguration.class,
+    super(TemplatesConfigurationVersion.v1_0,
+        com.devonfw.cobigen.impl.config.entity.io.v1_0.TemplatesConfiguration.class,
```

Comment:
The contextConfigurationUpgrader uses the impl.config class and not the generated from JAXB. We should determine the correct way and adjust the corresponding files.

Reply:
Can be adjusted and tested.

Reply:
Adjusted now to jaxb generated class.